roles/openshift_health_checker/openshift_checks/__init__.py

```diff
@@ -57,6 +65,14 @@ def subclasses(cls):
             for subclass in subclass.subclasses():
                 yield subclass
 
+    @staticmethod
```

Comment:
Yes.

You made me think about this again. I added it here in an attempt to be able to write `OpenShiftCheck.load_checks()` 
and facilitating imports as we talked elsewhere -- but in fact because of inheritance we're also making possible `SomeRandomCheck.load_checks()`.
That latter will likely crash, because this ugly thing here uses things like `__package__` and relative paths to `__file__`...

The reason it should not be in the top level as it was is to avoid the side-effect from simply importing this module...
It is better to explicitly load checks in the action plugin, and we don't need to "accidentally" load checks when we import this module in tests.

Let's possibly bring it back to the package level, we don't want this inherited in every check subclass.